src/nimblepkg/config.nim

```diff
@@ -23,8 +23,13 @@ proc initConfig(): Config =
   result.chcp = true
   result.cloneUsingHttps = true
   result.packageLists["official"] = PackageList(name: "Official", urls: @[
-    "https://raw.githubusercontent.com/nim-lang/packages/master/packages.json",
-    "https://nim-lang.org/nimble/packages.json"
```

Comment:
Though I would appreciate that "https://nim-lang.org/nimble/packages.json" is updated, which would greatly improve my user experience.

Reply:
Yep. GitHub sometimes stops working for me as well.

Reply:
I would also appreciate having "https://nim-lang.org/nimble/packages.json" as a fallback option because my connection to that github domain can be unreliable sometimes.